content/docs/atom-guide/look-dev/materials/material-type-file-spec.md

```diff
@@ -6,65 +6,479 @@ toc: true
+* **translateYProperty**: The name of a `float` type property. Controls the Y-translation.
+* **rotateDegreesProperty**: The name of a `float` type property. Controls the amount of rotation in degrees.
+* **float3x3ShaderInput**: The name of a `float3x3` shader constant for the final transform matrix.
+* **float3x3InverseShaderInput**: (optional) The name of a `float3x3` shader constant for an inverse of the final transform matrix. This may be needed by some shaders, for example ones that use a parallax mapping effect.
+
+Example JSON:
+```json
+    {
+        "type": "Transform2D",
+        "args": {
+            "transformOrder": [ "Rotate", "Translate", "Scale" ],
+            "centerProperty": "center",
+            "scaleProperty": "scale",
+            "scaleXProperty": "tileU",
+            "scaleYProperty": "tileV",
+            "translateXProperty": "offsetU",
+            "translateYProperty": "offsetV",
+            "rotateDegreesProperty": "rotateDegrees",
+            "float3x3ShaderInput": "m_uvMatrix",
+            "float3x3InverseShaderInput": "m_uvMatrixInverse"
+        }
+    }
+```
+
+#### ConvertEmissiveUnit
+
+This is a specialized functor for a standardized set of emissive lighting properties. It allows an enum property to select a light instensity unit (either nits or EV100) and converts a light intensity property's value accordingly.
+
+Arguments:
+* **intensityProperty**: The name of a `float` type property. Specifies the intensity of the emissive light.
+* **lightUnitProperty**: The name of a `Enum` type property. Indicates the units for the intesity value.
+* **shaderInput**: The name of a `float` shader constant that will receive the final intensity value. This is assumed to be in nits.
+* **ev100Index**: An integer value that corresponds to the enum value for EV100, according to the definition of the _lightUnitProperty_.
+* **nitIndex**: An integer value that corresponds to the enum value for nits, according to the definition of the _lightUnitProperty_.
+* **ev100MinMax**: An array of two floats, indicating the min and max intensity value when operating in _EV100_ mode.
+* **nitMinMax**: An array of two floats, indicating the min and max intensity value when operating in _nits_ mode.
+
+Example JSON:
+```json
+    {
+        "type": "ConvertEmissiveUnit",
+        "args": {
+            "intensityProperty": "intensity",
+            "lightUnitProperty": "unit",
+            "shaderInput": "m_emissiveIntensity",
+            "ev100Index": 0,
+            "nitIndex" : 1,
+            "ev100MinMax": [-10, 20],
+            "nitMinMax": [0.001, 100000.0]
+        }
+    },
+```
+
+#### HandleSubsurfaceScatteringParameters
+This is a specialized functor for a standardized set of properties for controlling subsurface scattering and light transmission features. See `Gems/Atom/Feature/Common/Assets/Materials/Types/Skin.materialtype` for example usage.
```

Comment:
Shortened file name and added link to the file in o3de repo. Revised last sentence so it's a complete sentence. 
```suggestion
This is a specialized functor for a standardized set of properties for controlling subsurface scattering and light transmission features. See [`Skin.materialtype`](https://github.com/o3de/o3de/blob/development/Gems/Atom/Feature/Common/Assets/Materials/Types/Skin.materialtype) for an example of its usage.
```